specification/security/get_service_accounts/types.ts

```diff
@@ -37,5 +36,5 @@ export class RoleDescriptor {
   applications?: ApplicationPrivileges[]
   metadata?: Metadata
   run_as?: string[]
-  transient_metadata?: Dictionary<string, UserDefinedValue>
+  transient_metadata?: TransientMetadata
```

Comment:
`TransientMetadata` is actually badly named and should be renamed to something like `TransientMetadataConfig` since its definition is

```ts
export class TransientMetadata {
  enabled: boolean
}
```

Reply:
Moved to `_types` as it was used 3 times and renamed to `TransientMetadataConfig `